crowbar_framework/app/views/barclamp/ipmi/_edit_attributes.html.haml

```diff
@@ -7,7 +7,12 @@
     = string_field :bmc_user
     = password_field :bmc_password
     = string_field :bmc_interface
-    = boolean_field :use_dhcp
-    = boolean_field :ignore_address_suggestions
+    = boolean_field :bmc_reconfigure,
+     "data-showit" => "true",
```

Comment:
What do these `data-showit` things do?

Reply:
They hide the parameters in reconfigure_container if reconfigure is disabled. The `use_dhcp` and `ignore_address_suggestions` make no sense if reconfiguration is not done.